### PR TITLE
Fix Eonar for LFR.

### PR DESCRIPTION
**Pull request by swordbeta**
- Update timers
- Need 3 life force
- No bats
- No rain of fel

Let me know if I have done anything wrong, first time helping out in a LUA mod! :)

### CODE REVIEW

**Comment by nebularg** [comment]
You should set lifeForceNeeded in the OnEngage as well in case someone runs LFR and another difficulty in the same session

**Comment by swordbeta** [comment]
@nebularg done

File: Antorus/Eonar.lua [hunk 1]
```diff
@@ -192,11 +210,12 @@ function mod:OnBossEnable()
 end
 
 function mod:OnEngage()
-	timers = self:Mythic() and timersMythic or self:Heroic() and timersHeroic or timersNormal
+	timers = self:Mythic() and timersMythic or self:Heroic() and timersHeroic or mod:LFR() and timersLFR or timersNormal
```
Comment (elvador) [inline Antorus/Eonar.lua]: Should be `self:LFR()`

File: Antorus/Eonar.lua [hunk 2]
```diff
@@ -205,7 +224,9 @@ function mod:OnEngage()
 	self:StartWaveTimer("bot", 1) -- Bottom wave spawns
 	self:StartWaveTimer("air", 1) -- Air wave spawns
```
Comment (elvador) [inline Antorus/Eonar.lua]: If there are no bats in LFR, we don't need to call this.

**Comment by swordbeta** [comment]
@elvador Nice catches, fixed. Let me know if I have to squash or if you guys squash when you merge.

**Comment by elvador** [comment]
Thanks!

**Pull request merged**